tests/phpunit/tests/rest-api/rest-widgets-controller.php

```diff
@@ -113,6 +113,10 @@ static function () {
 
```

Comment:
Move setting retrieval to within the `if` +
no need to check for the return value, just pass the `$default` parameter, like so:
```php
$settings = get_option( 'widget_testwidget', array() );
```

Props @binaryKitten

Reply:
Note: why wouldn't this return an array since it's registering it? Investigation needed as a confidence check of why this fix is needed. It does fix it. But why?

Reply:
I've updated the PR with the improvement as discussed during the livestream.

Marking this ready for review now.